test/try-delegate.wasm.fromBinary

```diff
@@ -0,0 +1,23 @@
+(module
```

Comment:
The original try-delegate.wasm file is this:
```wasm
try
  try 
  delegate 0
catch 0
  try
  delegate 0
end
```
This is to test whether targeting the depth 0 in a `try` body or `catch` body works, which is the innermost enclosing `try`. When parsing we create a block within each of `try` and `catch` body, but we shouldn't make those `delegate`s target those blocks because it will be a validation failure, so we have to remap those `delegate`s to the correct `try`.